dispatches/models/fossil_case/properties/test_thermaloil_properties.py

```diff
@@ -0,0 +1,223 @@
+#################################################################################
+# DISPATCHES was produced under the DOE Design Integration and Synthesis
+# Platform to Advance Tightly Coupled Hybrid Energy Systems program (DISPATCHES),
+# and is copyright (c) 2021 by the software owners: The Regents of the University
+# of California, through Lawrence Berkeley National Laboratory, National
+# Technology & Engineering Solutions of Sandia, LLC, Alliance for Sustainable
+# Energy, LLC, Battelle Energy Alliance, LLC, University of Notre Dame du Lac, et
+# al. All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. Both files are also available online at the URL:
+# "https://github.com/gmlc-dispatches/dispatches".
+#################################################################################
+
+###############################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+###############################################################################
+"""
+Tests for Thermal Oil property package.
+Authors: Naresh Susarla
+"""
+
+import pytest
+from pyomo.environ import (ConcreteModel,
+                           Constraint,
+                           Expression,
+                           Param,
+                           value,
+                           Var)
+from pyomo.util.check_units import assert_units_consistent
+from idaes.core import (MaterialBalanceType,
+                        EnergyBalanceType)
+from idaes.core.util import get_solver
+import idaes.logger as idaeslog
+import thermaloil_properties
+
+
+# -----------------------------------------------------------------------------
+# Get default solver for testing
+solver = get_solver()
```

Comment:
I vaguely remember Dan mentioning this. It is not recommended to have code (other than imports) outside functions. He said that either put the code inside the functions or in `if __name__ == "__main__:`
So, if possible, please move this line inside the function where it is needed.

Reply:
based on the similar comment below, left unchanged